client/pyroclient/client.py

```diff
@@ -26,7 +26,7 @@
     #################
     # DETECTIONS
     #################
-    "detections-create": "/detections",
+    "detections-create": "/detections/",
```

Comment:
this shouldn't be a problem :thinking: 
Could you revert this and try to find the problem? I'm supposed to have solved it in #317

Reply:
Besides this is unrelated to the PR :sweat_smile:

Reply:
I had a 307 redirection in the log, not really an issue but still